blog/tag/seo.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: tag
+tag: seo
+permalink: /blog/tag/seo/
+redirect_from:
```

Comment:
Baidu prevails in China, and Yahoo in Russia. So, statement like that will only be true depending on what market is targeted. Surely it applies to US.